src/ansys/mapdl/core/mapdl.py

```diff
@@ -4431,3 +4483,48 @@ def _parse_rlist(self):
                 const_[set_][jlimit] = values_[i]
 
         return const_
+
```

Comment:
there are quite a few `_parse_*list` functions in the mapdl class.  Maybe its time to add a new `list_parser.py` with these as free functions?

Reply:
I kind of agree. I will need a bit more details about what you mean with `list_parser` is this just another module which hold this functions? (Not sure if all are static)

Reply:
@germa89 yeah just a new module to hold the functions. Instead of `self` the first argument to these functions would be `mapdl`

Reply:
It is a good idea. Creating issue.

https://github.com/ansys/pymapdl/issues/2159